spring-integration-ftp/src/main/resources/org/springframework/integration/ftp/config/spring-integration-ftp-5.0.xsd

```diff
@@ -577,6 +577,19 @@
 					]]></xsd:documentation>
 					</xsd:annotation>
 				</xsd:attribute>
+				<xsd:attribute name="scanner" type="xsd:string">
```

Comment:
Well, that's wrong place for this attribute.
This `base-inbound-adapter-type` is used from the `inbound-streaming-channel-adapter` as well, which doesn't have any local work by nature.